src/node/utils/ExportHtml.js

```diff
@@ -487,6 +487,7 @@ function getHTMLFromAtext(pad, atext, authorColors)
 exports.getPadHTMLDocument = async function (padId, revNum)
 {
   let pad = await padManager.getPad(padId);
+  const padName = padId.split(":").pop()
```

Comment:
What is padName to padId?  This is confusing.   Pad Name is what I would call the "Pad Title" IE what's used in ep_title or whatever it's called.

Reply:
You right, the `padName` can be change to `padTitle`.

Reply:
I mean, it's still not explained what it is.. .   IE include docs..  But is padTitle the same as a padId ?  or is it a nestedPadId?!  It's really confusing.. Some inline comments maybe?

Reply:
Oh, i got it what you mean, let me put it in this way for an explanation(also i'll add some comments there later)

The router like this: `/p/pad1/pad2/pad3`
The `padId` will be `pad1:pad2:pad3`
and the `padName` will be `pad3`, which is always the last part of padId.

Reply:
the padName that is relative to what?  pad1 or pad2 or both?

if your padId is pad1:pad2:pad3 I imagine it will break a lot of things but I could be wrong..  Like I said getSub uses ``:`` delims